deis-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -17,4 +17,5 @@ data: {{ if eq .storage "gcs" }}
   accesskey: {{ b64enc .s3.accesskey }}
   secretkey: {{ b64enc .s3.secretkey }}
   region: {{ b64enc .s3.region }}
-  bucket: {{ b64enc .s3.bucket }}{{ end }}
+  bucket: {{ b64enc .s3.bucket }}
+  database-bucket: {{ b64enc .s3.database-bucket }}{{ end }}
```

Comment:
what's the difference between bucket and database-bucket? Can't we store registry and the database's information in the same bucket through path separation?